gradle.properties

```diff
@@ -5,7 +5,7 @@ dependency_repo=libs-release
 buildInfo.licenseControl.runChecks=true
 buildInfo.licenseControl.autoDiscover=true
 buildInfo.build.name=com.dell.isg.smi:service-virtualidentity
-systemProp.sonar.host.url=http://100.68.126.201:9000
-#systemProp.sonar.host.url=https://sonarqube.com
-#systemProp.sonar.organization=rackhd-smi
-#systemProp.sonar.login=##sonarqubeLogin-placeholder##
+#systemProp.sonar.host.url=http://100.68.126.201:9000
```

Comment:
Remove unused code?

Reply:
The code is not unused code which will play important role in cicd pipeline.